portal-api/src/v1/gef/controllers/clone-application.controller.js

```diff
@@ -78,6 +78,8 @@ const cloneFacilities = async (currentDealId, newDealId) => {
       allFacilities[val].updatedAt = Date.now();
       // reset the ukefFacilityId
       allFacilities[val].ukefFacilityId = null;
+      // reset the coverDateConfirmed
+      allFacilities[val].coverDateConfirmed = null;
```

Comment:
Can we test this?

Reply:
this is going to be tricky because the deal has to be accepted by ukef (with or without conditions). 
The problem is that when you submit the deal from GEF to TFM, you have to wait for the number generator to finish and for the statuses to get updated. Otherwise if I log in immediately (which is what the e2e-tests will do), the status will get overwritten

Reply:
I was thinking unit test at least?